examples/step-50/doc/results.dox

```diff
@@ -1,98 +1,312 @@
-   14 CG iterations needed to obtain convergence.
-Cycle 7:
-   Number of active cells:       3194
-   Number of degrees of freedom: 3603 (by level: 8, 25, 89, 328, 1032, 2200, 1392)
-   16 CG iterations needed to obtain convergence.
-</pre>
-That's not perfect &mdash; we would have hoped for a constant number
-of iterations rather than one that increases as we get more and more
-degrees of freedom &mdash; but it is also not far away. The reason for
-this is easy enough to understand, however: since we have a strongly
-varying coefficient, the operators that we assembly by quadrature on
-the lower levels become worse and worse approximations of the operator
-on the finest level. Consequently, even if we had perfect solvers on
-the coarser levels, they would not be good preconditioners on the
-finest level. This theory is easily tested by comparing results when
-we use a constant coefficient: in that case, the number of iterations
-remains constant at 9 after the first three or four refinement steps.
-
-We can also compare what this program produces with how @ref step_5
-"step-5" performed. To solve the same problem as in step-5, the only
-two changes that are necessary are (i) to replace the body of the
-function <code>LaplaceProblem::refine_grid</code> by a call to
-<code>triangulation.refine_global(1)</code>, and (ii) to use the same
-SolverControl object and tolerance as in step-5 &mdash; the rest of the
-program remains unchanged. In that case, here is how the solvers used
-in step-5 and the multigrid solver used in the current program
-compare:
-<table align="center">
-<tr><th>cells</th><th>step-5</th><th>step-16</th></tr>
-<tr><td>20</td>   <td>13</td> <td>6</td> </tr>
-<tr><td>80</td>   <td>17</td> <td>7</td> </tr>
-<tr><td>320</td>  <td>29</td> <td>9</td> </tr>
-<tr><td>1280</td> <td>51</td> <td>10</td> </tr>
-<tr><td>5120</td> <td>94</td> <td>11</td> </tr>
-<tr><td>20480</td><td>180</td><td>13</td></tr>
+.
+.
+.
+@endcode
+Here, the timing of the `solve()` function is spilt up in 3 parts: setting
+up the multigrid preconditioner, execution of a single multigrid vcycle, and
+the CG solver. The vcycle that is timed is unnecessary for the overall solve
+and only meant to give an insight at the different costs for AMG and GMG.
+Also it should be noted that when using the AMG solver, ``Workload imbalance''
+is not included in the output since the hierarchy of coarse meshes are not
+required.
+
+In addition to the AMG and GMG solvers in this tutorial, included will be timings
+from a 3rd matrix-free (MF) GMG solver on the same problem (see possible extensions
+for a discussion on what is required for the matrix-free solver). We will refer to
+the GMG solver in tutorial as the matrix-based (MB) GMG solver.
+
+The following table gives the timings for setup, assembly, and solve for GMG and AMG
+on up to 256M DoFs and 7168 processors.
+<table align="center" class="doxtable">
+<tr>
+  <th></th>
+  <th colspan="1"></th>
```

Comment:
I think that this line should be removed. The titles do not match the data of the table when I compile it with Doxygen.

Reply:
Yes, I've removed the line.